scripts/us_census/pep/monthly_population_estimate/preprocess.py

```diff
@@ -0,0 +1,482 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" A Script to process USA Census PEP monthly population data
+    from the datasets in provided local path.
+    Typical usage:
+    1. python3 preprocess.py
+    2. python3 preprocess.py -i input_data
+"""
+import os
+import re
+import sys
+
+import numpy as np
+import pandas as pd
+from absl import app
+from absl import flags
+
+pd.set_option("display.max_columns", None)
+
+FLAGS = flags.FLAGS
+default_input_path = os.path.dirname(
+    os.path.abspath(__file__)) + os.sep + "input_data"
+
+if ("ip_path" in sys.argv and len(sys.argv) > 2):

```

Comment:
I don't understand why we need ip_path.  Given that you're using absl library, please use flags.

For example: https://github.com/datacommonsorg/data/blob/master/scripts/glims/rgi/process.py#L121-L129

Reply:
ip path has been removed and a default value has been set for the flag